spotifycli/spotifycli.py

```diff
@@ -82,32 +85,37 @@ def start_shell():
 def add_arguments():
     parser = argparse.ArgumentParser(description=__doc__)
     for argument in get_arguments():
-        parser.add_argument(argument[0], help=argument[1], action="store_true")
+        if not argument[2]:
+            parser.add_argument(argument[0], help=argument[1], action="store_true")
+        else:
+            parser.add_argument(argument[0], help=argument[1], action="store")
     parser.add_argument("--client", action="store", dest="client",
                         help="sets client's dbus name", default="spotify")
     return parser.parse_args()
 
 
 def get_arguments():
     return [
-        ("--version", "shows version number"),
-        ("--status", "shows song name and artist"),
-        ("--statusposition", "shows song name and artist, with current playback position"),
-        ("--statusshort", "shows status in a short way"),
-        ("--song", "shows the song name"),
-        ("--songshort", "shows the song name in a short way"),
-        ("--artist", "shows artist name"),
-        ("--artistshort", "shows artist name in a short way"),
-        ("--album", "shows album name"),
-        ("--position", "shows song position"),
-        ("--arturl", "shows album image url"),
-        ("--playbackstatus", "shows playback status"),
-        ("--play", "plays the song"),
-        ("--pause", "pauses the song"),
-        ("--playpause", "plays or pauses the song (toggles a state)"),
-        ("--lyrics", "shows the lyrics for the song"),
-        ("--next", "plays the next song"),
-        ("--prev", "plays the previous song")
+        ("--version", "shows version number", False),
+        ("--status", "shows song name and artist", False),
+        ("--statusposition", "shows song name and artist, with current playback position", False),
+        ("--statusshort", "shows status in a short way", False),
+        ("--song", "shows the song name", False),
+        ("--songshort", "shows the song name in a short way", False),
+        ("--artist", "shows artist name", False),
+        ("--artistshort", "shows artist name in a short way", False),
+        ("--album", "shows album name", False),
+        ("--position", "shows song position", False),
+        ("--arturl", "shows album image url", False),
+        ("--playbackstatus", "shows playback status", False),
+        ("--play", "plays the song", False),
+        ("--pause", "pauses the song", False),
+        ("--playpause", "plays or pauses the song (toggles a state)", False),
+        ("--lyrics", "shows the lyrics for the song", False),
+        ("--next", "plays the next song", False),
+        ("--prev", "plays the previous song", False),
+        ("--songuri", "plays the track at the provided Uri", True),
+        ("--listuri", "plays the playlist at the provided Uri", True),
```

Comment:
Added a third argument to each of the entries in the list which specified whether the arg will specify a value or not.  All the existing entries here should just continue to trigger a `true` value to be stored.  But in the case of the two new ones, the user will be expected to provide a value after the argument.  So those have a `True` in the argument.

I could have done this many other ways, but this seems the most intentional.  I'm certainly open to your thoughts.